tests/sentry/attachments/test_redis.py

```diff
@@ -22,21 +22,9 @@ def mock_client():
     return FakeClient()
 
 
-@pytest.fixture(params=["rb", "rediscluster"])
+@pytest.fixture(params=["rediscluster"])
 def mocked_attachment_cache(request, mock_client):
-    class RbCluster:
-        def get_routing_client(self):
-            return mock_client
-
-    if request.param == "rb":
-        with mock.patch(
-            "sentry.cache.redis.get_cluster_from_options", return_value=(RbCluster(), {})
-        ) as cluster_get:
-            attachment_cache = import_string("sentry.attachments.redis.RbAttachmentCache")(hosts=[])
-            cluster_get.assert_any_call("SENTRY_CACHE_OPTIONS", {"hosts": []})
-            assert isinstance(attachment_cache.inner, RbCache)
-
-    elif request.param == "rediscluster":
+    if request.param == "rediscluster":
```

Comment:
could probabl remove the `params=` and this `if` statement too since there's only one way now!

Reply:
Definitely true. Updated.